wire-ios/Wire-iOS/Sources/UserInterface/Settings/DeviceManagement/ClientListViewController.swift

```diff
@@ -71,6 +71,8 @@ final class ClientListViewController: UIViewController,
     private let clientFilter: (UserClient) -> Bool
     private let userSession: UserSession?
     private let contextProvider: ContextProvider?
+    private var deviceInfoViewModel: DeviceInfoViewModel?
```

Comment:
why `deviceInfoViewModel` is optional?

Reply:
We do not need this info to display the Client list. This is holding a reference to the the details view which is displayed upon tapping a client.